modules/virt-creating-vm-instancetype.adoc

```diff
@@ -58,23 +62,57 @@ Logos of operating systems that are not available in the cluster are shown at th
 +
 In addition, there is a link to the *Create a Windows boot source* quick start. The same link appears in a popover if you hover the pointer over the question mark icon next to the _Select volume to boot from_ line.
 +
-Immediately after you install the environment or when the environment is disconnected, the list of volumes to boot from is empty. In that case, three operating system logos are displayed: Windows, {op-system-base}, and Linux. You can add a new volume that meets you requirements by clicking the *Add volume* button.
+Immediately after you install the environment or when the environment is disconnected, the list of volumes to boot from is empty. In that case, three operating system logos are displayed: Windows, {op-system-base}, and Linux. You can add a new volume that meets your requirements by clicking the *Add volume* button.
+endif::[]
+
+ifndef::dynamic-key,virt-create-vms[]
+ Select a suitable bootable volume from the list. If the list is truncated, click the *Show all* button to display the entire list.
++
+[NOTE]
+====
+The bootable volume table lists only those volumes in the `openshift-virtualization-os-images` namespace that have the `instancetype.kubevirt.io/default-preference` label.
+====
+
+** Optional: Click the star icon to designate a bootable volume as a favorite. Starred bootable volumes appear first in the volume list.
```

Comment:
Missing the start of a list item here. Should be `.` for ordered now that it's not a 2-option choice for this conditional. Also needs the missing `endif::[]` for the `ifndef`:
```suggestion
. Select a suitable bootable volume from the list. If the list is truncated, click the *Show all* button to display the entire list.
+
[NOTE]
====
The bootable volume table lists only those volumes in the `openshift-virtualization-os-images` namespace that have the `instancetype.kubevirt.io/default-preference` label.
====

.. Optional: Click the star icon to designate a bootable volume as a favorite. Starred bootable volumes appear first in the volume list.
endif::[]
```